TTGSnackbar/TTGSnackbar.swift

```diff
@@ -849,45 +849,45 @@ private extension TTGSnackbar {
         separateView.backgroundColor = separateViewBackgroundColor
         contentView.addSubview(separateView)
 
-        activityIndicatorView = UIActivityIndicatorView(activityIndicatorStyle: .white)
+        activityIndicatorView = UIActivityIndicatorView(style: .white)
         activityIndicatorView.translatesAutoresizingMaskIntoConstraints = false
         activityIndicatorView.stopAnimating()
         contentView.addSubview(activityIndicatorView)
 
         // Add constraints
         let hConstraints = NSLayoutConstraint.constraints(
             withVisualFormat: "H:|-0-[iconImageView]-2-[messageLabel]-2-[seperateView(0.5)]-2-[actionButton(>=44@999)]-0-[secondActionButton(>=44@999)]-0-|",
-            options: NSLayoutFormatOptions(rawValue: 0),
+            options: NSLayoutConstraint.FormatOptions(rawValue: 0),
```

Comment:
nab, there is no need to do `NSLayoutFormatOptions(rawValue: 0)`. rather just type in `[]`